apps/glusterfs/volume_entry.go

```diff
@@ -959,6 +963,15 @@ func (v *VolumeEntry) cloneVolumeRequest(db wdb.RODB, clonename string) (*execut
 	vcr.Volume = v.Info.Name
 	vcr.Clone = clonename
 
+	var sshhost string
+	sshhost, err := v.GetManageHost(db)
+	if err != nil {
+		return nil, "", err
+	}
+	return vcr, sshhost, nil
+}
+
+func (v *VolumeEntry) GetManageHost(db wdb.RODB) (string, error) {
```

Comment:
this function can probably be private? Just `getManageHost()` would do in that case.